### PR TITLE
onboardingUpdateV2

### PR DESCRIPTION
**Pull request by AndreasInk**
- Added the powered by Covid Watch icon
- Fixed spacing issues with smaller devices

### CODE REVIEW

**Comment by zssz** [comment]
Hi @AndreasInk, before merging this in please make these changes:
1. Remove print statements.
2. Remove powered by Covid Watch from Onboarding screens, which you added.
3. Reformat code with Xcode to be aligned with the style used in the other parts of the project.
4. Replace the hard-coded number of pages (3) with a dynamic one.

**Comment by AndreasInk** [comment]
Hi! I can do that, thank you for the feedback @zssz

**Comment by AndreasInk** [comment]
I made the following updates @zssz
1. Removed print statements 
2. Removed powered by Covid Watch icon from onboarding 
3. Reformatted the code with control i (removed weird indents and whitespaces)
4. Created a dynamic loop when the user presses the arrow buttons in onboarding  (no longer set to not progress when page 4 is presented)

File: CovidWatch/Views/Support/PageView.swift [hunk 1]
```diff
@@ -1,64 +1,68 @@
...
-        
+                HStack(spacing: .standardSpacing * 20){
+                    Button(action: {
+                        
+                        if self.currentPage != 0 {
+                            self.currentPage -= 1
+                        } else {
+                            self.currentPage = self.viewControllers.count - 1
+                        }
+
```
Comment (zssz) [inline CovidWatch/Views/Support/PageView.swift:36]: Please remove the many extra new lines here.

**Comment by zssz** [comment]
@AndreasInk, can you make the following changes?
1. Remove the extra padding in the bottom, see Screenshot 2020-06-29 at 18 23 25
2. There are still lots of extra new lines. I issued a separate comment.

**Comment by AndreasInk** [comment]
I fixed the following @zssz...
1) Removed extra padding at the bottom of onboarding
2) Replaced extra padding with .vertical .standardSpacing

Still need to...
1) Remove extralines in code

**Comment by AndreasInk** [comment]
I fixed the following @zssz...

1) Removed extra lines of code in PageView

Still need to...

1) Add SwiftLint

**Comment by AndreasInk** [comment]
I added the following @zssz...
1) SwiftLint
2) Addressed most SwiftLint errors in PageView

**Comment by zssz** [comment]
Can you put the SwiftLint feature in a separate PR? In that PR please make sure that SwiftLint doesn't show an errors while building the project. This means you have to modify the formatting style of some of the source files.

Regarding this PR, I can approve it, once you take SwiftLint out.

**Comment by AndreasInk** [comment]
Sounds good, I’ll do that

**Comment by AndreasInk** [comment]
SwiftLint is removed @zssz, next I'll create a new pull request with SwiftLint and fix the errors SwiftLint displays.

**Comment by zssz** [comment]
@AndreasInk, sorry for the slow response. Can you add a [BlurView](https://github.com/covidwatchorg/covidwatch-ios-en/blob/92d08cc27730d9b1c16295b9bdc7f94ac6638611/CovidWatch/Views/Support/Blur.swift) behind the page control? Make sure it floats above the content (similar to the header bar) so that on the last screen the user knows that there's content below it (unlike in the screenshot below).

Screenshot 2020-07-02 at 16 45 13

**Comment by AndreasInk** [comment]
You're all good @zssz, I can definitely add that.

**Comment by AndreasInk** [comment]
Hi @zssz! I added the blur view to PageView, please let me know the next steps or anything that needs to be fixed.

**Comment by zssz** [comment]
Hey! The PR has merge conflicts. Can you fix those? Thanks.

**Pull request closed without merging**